rasa/engine/runner/__init__.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
```

Comment:
I'd prefer to move this a named module instead of having it in `__init__` . I think `__init__` should be empty except maybe some imports